charts/jaeger-operator/templates/deployment.yaml

```diff
@@ -48,10 +48,12 @@ spec:
           - containerPort: 9443
             name: webhook-server
             protocol: TCP  
+          {{- if .Values.certs.certificate.create }}
```

Comment:
Check out the https://github.com/jaegertracing/helm-charts/tree/main/charts/jaeger-operator#prerequisites for the operator chart and my comment in conversation. In case you don't want to use cert-manager you should create certificate for webhook service in a secret instead.

Reply:
@czomo thanks for the explanation. I have already configured it to use webhooks and self-signed issuer/certs now so am good. 
Basically it started with if we can disable both webhooks and hence certificates for webhooks. 
Like what if I disable both the webhooks ? (https://github.com/jaegertracing/helm-charts/blob/jaeger-operator-2.32.2/charts/jaeger-operator/values.yaml#L22)
Why should I still care about certificates ?

Reply:
jaeger-operator can be deployed in [multiple ways](https://www.jaegertracing.io/docs/1.24/operator/#install-modes), that's why there is option to disable specific components.

webhooks in general are integrated part of jaeger because they are responsible for validating jeager resources(validating webhook) or in case of mutating webhooks adding jaeger sidecars. However there are some cases when you want do disable them because of  conflicts with other resources.  Be carefull about that.
One of the purpose of certs is handling secure connection between service and webhooks, more features will be added in the future.

Reply:
Ok so I think its wiser to not disable any of that :) We don't this PR anymore in that case.

Reply:
We should be able to disable creating the certificates so people can use their own `cert-manager` issuers and settings. Is this PR being merged?